Bixal/move.mil
# remove body field from basic page content type

Third time's the charm?

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Removes the body field on (and only on) the basic page content type, including its edit and display views
- Note that the teaser display view is now empty, but we're not using it anyway so don't worry about that

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Sync latest staging db and files if testing locally
1. Import config
1. Check Entitlements and Service-Specific pages in particular to make sure the intro text/images are there

## Screenshots

Local with basic page body field removed:
<img width="961" alt="screen shot 2018-06-12 at 2 15 17 pm" src="https://user-images.githubusercontent.com/29130580/41308876-19341b80-6e4b-11e8-86b5-d0cd7b8893e1.png">
